core/src/Kokkos_Atomics_Desul_Volatile_Wrapper.hpp

```diff
@@ -156,4 +157,5 @@ T atomic_compare_exchange(volatile T* const dest, const T compare, const T desir
 
 }
 // clang-format on
+#endif  // KOKKOS_ENABLE_IMPL_DESULA_TOMICS
```

Comment:
O_o

Reply:
?

Reply:
I needed this only for the self sufficiency test of Kokkos on Windows ...